agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -597,6 +601,8 @@ class backhaul_manager : public btl::transport_socket_thread {
 
     sExpectedChannelSelection m_expected_channel_selection;
 
+    bool backhaul_sta_steering_enable = false;
```

Comment:
This change is unrelated and not needed I think.

Reply:
Moved to commit with usage.